share/github-backup-utils/ghe-backup-config

```diff
@@ -417,7 +417,7 @@ is_service_external(){
     "mysql")
       if [ -n "$config_file" ]; then
         enabled=$(GIT_CONFIG="$config_file" git config mysql.external.enabled)
-        [ "$enabled" == "true" ];
+        [ "$enabled" = true ];
```

Comment:
I think this change is functionally the same as what we already have; `=` is just the POSIX compliant version of `==` and the quotes aren't doing much on the right hand operator here since it's only a single word.

Reply:
during my testing, I got following if `$enabled=true`
```
zsh: = not found
```